src/types.ts

```diff
@@ -53,6 +81,86 @@ export interface FrameStartType {
   sceneFrameCounter: number | null;
 }
 
+export interface GameInfoType {
+  gameBitfield1: number | null;
+  gameBitfield2: number | null;
+  gameBitfield3: number | null;
+  gameBitfield4: number | null;
+  bombRainEnabled: boolean | null;
+  itemSpawnBehavior: number | null;
+  selfDestructScoreValue: number | null;
+  startingTimerFrames: number | null;
+  itemSpawnBitfield1: number | null;
+  itemSpawnBitfield2: number | null;
+  itemSpawnBitfield3: number | null;
+  itemSpawnBitfield4: number | null;
+  itemSpawnBitfield5: number | null;
+  damageRatio: number | null;
+}
+
+export enum TimerType {
+  NONE = 0b00,
+  DECREASING = 0b10,
+  INCREASING = 0b11,
+}
+
+export enum ItemSpawnBehaviorType {
+  OFF = 0xff,
+  VERY_LOW = 0x00,
+  LOW = 0x01,
+  MEDIUM = 0x02,
+  HIGH = 0x03,
+  VERY_HIGH = 0x04,
+}
+
+export enum EnabledItemType {
```

Comment:
```suggestion
export enum ItemType {
```
Can we just call this `ItemType`?

Reply:
There is a distinction to be made between game items and enabled items in the settings.

In my upcoming Stadium computers, the in-game item's type is used to pick out the targets from the game data. We could also use it to give items names based on their type, such as lasers and turnips.

I'm fine with changing this to `ItemType` if needed.